grabbit/src/main/groovy/com/twcable/grabbit/ClientJobStatus.groovy

```diff
@@ -82,4 +100,12 @@ class ClientJobStatus {
             get(explorer, id)
         }
     }
+
+    static List<ClientJobStatus> getAllForTransaction(@Nonnull final JobExplorer explorer, @Nonnull final long transactionID) {
+        if(!explorer) throw new IllegalArgumentException("jobExplorer == null")
+        if(!transactionID) throw new IllegalArgumentException("transactionID == null")
```

Comment:
Are there any restrictions on `transactionID`? Does it have to be strictly positive ( >= 1 )?